### PR TITLE
Dar 390 contact support

### PR DESCRIPTION
**Pull request by jstirnaman**
Closes https://github.com/influxdata/DAR/issues/390
> We have a new base annual plan for Serverless (and Cloud2) users that includes support. With the introduction of this plan, we are restricting support access for regular users on usage-based plans. There are a few places we tell users to contact support in the Serverless docs that need to be updated to avoid support requests and EARs...

Removes Support contact instructions for operations where it's no longer needed or provided.
Updates the footer to be consistent with support model changes.
Suggests alternatives to deleting data.

### CODE REVIEW

File: content/influxdb/cloud-serverless/reference/internals/storage-limits.md [hunk 1]
```diff
@@ -48,9 +49,7 @@ number of columns allowed in a table.
 
 #### Potential solutions
 
-- Consider storing new fields in a new measurement.
-- [Contact InfluxData Support](https://support.influxdata.com) to delete any
-  unnecessary data.
+Consider storing new fields in a new [measurement](/influxdb/cloud-serverless/reference/glossary/#measurement).
```
Comment (btasker) [inline content/influxdb/cloud-serverless/reference/internals/storage-limits.md:52]:
```suggestion
Consider storing new fields in a new [measurement](/influxdb/cloud-serverless/reference/glossary/#measurement) (but see note below about the measurement/table limits that apply).
```

File: content/influxdb/cloud-serverless/admin/billing/_index.md [hunk 2]
```diff
@@ -134,5 +130,5 @@ Billing occurs on the first day of the month for the previous month. For example
...
-| **10-14 days later**         | Account completely disabled. During this period, you must contact us at [support.influxdata.com](https://support.influxdata.com) to process your payment and enable your account.                                                 |
-| **21 days later**            | Account suspended. Contact [support.influxdata.com](https://support.influxdata.com) to settle your final bill and retrieve a copy of your data or access to InfluxDB Cloud dashboards, tasks, Telegraf configurations, and so on. |
+| **10-14 days later**         | Account completely disabled. During this period, you must contact [InfluxData Support](https://support.influxdata.com/) to process your payment and enable your account.                                                 |
+| **21 days later**            | Account suspended. Contact [InfluxData Support](https://support.influxdata.com/) to settle your final bill and retrieve a copy of your data or access to InfluxDB Cloud dashboards, tasks, Telegraf configurations, and so on. |
```
Comment (jstirnaman) [inline content/influxdb/cloud-serverless/admin/billing/_index.md:134]:
```suggestion
| **21 days later**            | Account suspended. Contact [InfluxData Support](https://support.influxdata.com/) to settle your final bill and retrieve a copy of your data. |
```
@btasker @M617263Kaplan Does Support provide this for Serverless?

**Review by btasker — APPROVED** [review]
LGTM - a couple of changes suggested, but by no means required.

File: content/influxdb/cloud/influxdb-templates/monitor-enterprise.md [hunk 3]
```diff
@@ -181,4 +181,4 @@ Send a notification to PagerDuty or HTTP endpoints (other webhooks) by [upgradin
 
 For InfluxDB Enterprise customers, Insights and Aware are free services that can monitor your data. InfluxDB Insights sends your data to a private Cloud account and will be monitored with the help of the support team. InfluxDB Aware is a similar service, but you monitor your data yourself.
```
[the remark this answers is not in the document]
Comment (jstirnaman) [reply content/influxdb/cloud/influxdb-templates/monitor-enterprise.md:182]: > Should actually take the mention of Aware out. Doesn't really fit the scope of this PR though, I'll raise an additional one in a bit for that

Thanks, I meant to ask about this paragraph.

File: content/influxdb/cloud-serverless/reference/internals/storage-limits.md (hunk 1 above)
Comment (jstirnaman) [inline content/influxdb/cloud-serverless/reference/internals/storage-limits.md]:
```suggestion
Consider storing new fields in a new [measurement](/influxdb/cloud-serverless/reference/glossary/#measurement) (not to exceed the [maximum number of measurements (tables)](#maximum-number-of-tables-reached)).
```

File: content/influxdb/cloud/influxdb-templates/monitor-enterprise.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (btasker) [reply content/influxdb/cloud/influxdb-templates/monitor-enterprise.md:182]: :+1: raised a PR at https://github.com/influxdata/docs-v2/pull/5005

File: content/influxdb/cloud-serverless/reference/internals/storage-limits.md (hunk 1 above)
Comment (btasker) [reply content/influxdb/cloud-serverless/reference/internals/storage-limits.md]: LGTM

File: content/influxdb/cloud-serverless/reference/internals/storage-limits.md [hunk 4]
```diff
@@ -48,9 +49,9 @@ number of columns allowed in a table.
 
 #### Potential solutions
 
-- Consider storing new fields in a new measurement.
-- [Contact InfluxData Support](https://support.influxdata.com) to delete any
-  unnecessary data.
+- Consider storing new fields in a new [measurement](/influxdb/cloud-serverless/reference/glossary/#measurement) (not to exceed the [maximum number of tables](#maximum-number-of-tables-reached)).
+- Review [InfluxDB schema design recommendations](/influxdb/cloud-serverless/write-data/best-practices/schema-design/).
+- Customers with an annual or support contract can [contact InfluxData Support](https://support.influxdata.com) to request a review of their database schema.
```
Comment (jstirnaman) [inline content/influxdb/cloud-serverless/reference/internals/storage-limits.md:54]:
```suggestion
- Customers with an annual or support contract can contact [InfluxData Support](https://support.influxdata.com) to request a review of their database schema.
```

**Pull request merged**